test/common/odp_cunit_common.h

```diff
@@ -15,8 +15,11 @@
 
 #include <stdint.h>
 #include <inttypes.h>
-#include "CUnit/Basic.h"
-#include "CUnit/TestDB.h"
+#include <CUnit/Basic.h>
+#include <CUnit/TestDB.h>
+#if TEST_CUNIT_XML
+#include <CUnit/Automated.h>
+#endif
```

Comment:
No need to hide it under `#if`

Reply:
our arm tests use some previous version where function declared as finc(); i.e. not func(void) and gcc fails on it. In task  which run inside Shippable it's corrected.

Reply:
@muvarov then it's better to update CUnit version on target, rather than workaround its bugs.